src/commands/baseTypeConverters.ts

```diff
@@ -77,15 +77,15 @@ export const baseTypeConverters = {
     }
 
     const guild = message.channel.guild;
-    const channel = guild.channels.cache.get(channelId as Snowflake);
+    const channel = guild.channels.cache.get(channelId);
     if (!channel) {
       throw new TypeConversionError(`Could not find channel for channel id \`${channelId}\``);
     }
 
     return channel;
   },
 
-  textChannel(value: string, { message }: CommandContext<AnyPluginData<any>>): TextChannel {
+  textChannel(value: string, { message }: CommandContext<AnyPluginData<any>>): TextChannel & ThreadChannel {
```

Comment:
Seems like a mistake
```suggestion
  textChannel(value: string, { message }: CommandContext<AnyPluginData<any>>): TextChannel | ThreadChannel {
```

Reply:
Not only does this sound like a mistake, it also smells like one :D Thank you